### PR TITLE
Convert some simple tests to multi-database tests

### PR DESCRIPTION
**Pull request by thijs-obj**
Just converted some tests to multi-database tests.
Please have a look when you have time @KathiaBarahona

### CODE REVIEW

File: bach/tests/functional/bach/test_df_astype.py [hunk 1]
```diff
@@ -123,16 +130,3 @@ def test_astype_to_json():
             [3, [{"_type": "WebDocumentContext", "id": "#document"}, {"_type": "SectionContext", "id": "home"}, {"_type": "SectionContext", "id": "top-10"}, {"_type": "ItemContext", "id": "5o7Wv5Q5ZE"}]]
         ]
     )
-
-
-def test_astype_dtype_aliases():
```
Comment (thijs-obj) [inline bach/tests/functional/bach/test_df_astype.py:128]: Moved this to the unittest folder

File: bach/tests/functional/bach/test_df_materialize.py [hunk 2]
```diff
@@ -37,13 +39,21 @@ def test_materialize(inplace: bool):
...
     # has been moved to the new underlying base_node.
-    for series in bt.data.values():
-        assert series.expression.to_sql(dialect) != f'"{series.name}"'
-    for series in bt_materialized.data.values():
-        assert series.expression.to_sql(dialect) == f'"{series.name}"'
+    for series_name in bt.data_columns:
+        if is_postgres(dialect):
+            expected_to_sql = f'"{series_name}"'
+        elif is_bigquery(dialect):
+            # TODO: This path is never taken, as we only test with Postgres dialect/engine.
```
Comment (thijs-obj) [inline bach/tests/functional/bach/test_df_materialize.py:50]: Updated this test, then found out it didn't work. Didn't want to through this away now

File: bach/tests/functional/bach/test_df_sort_values.py [hunk 3]
```diff
@@ -70,8 +73,11 @@ def test_sort_values_parameters():
         )
 
 
-def test_sort_values_wrong_parameters():
-    bt = get_bt_with_test_data(full_data_set=True)
+# TODO: below are unit tests, move them to tests/unit/bach/test_df_sort_values.py
```
Comment (thijs-obj) [inline bach/tests/functional/bach/test_df_sort_values.py:76]: Will do that in the next PR. As changing and moving code in one step makes it a harder to review the changes

Comment (thijs-obj) [reply bach/tests/functional/bach/test_df_sort_values.py:76]: https://github.com/objectiv/objectiv-analytics/pull/579

**Comment by thijs-obj** [comment]
Ready for review. Please have a look @KathiaBarahona

**Review by KathiaBarahona — APPROVED** [review]

**Pull request merged**